compose/cli/docker_client.py

```diff
@@ -32,4 +32,4 @@ def docker_client():
         )
 
     timeout = int(os.environ.get('DOCKER_CLIENT_TIMEOUT', 60))
-    return Client(base_url=base_url, tls=tls_config, version='1.18', timeout=timeout)
+    return Client(base_url=base_url, tls=tls_config, version='1.19', timeout=timeout)
```

Comment:
I'd just like to add that I believe the variable name is more accurate and the placement of the environ.get is in a better spot in [#1780](https://github.com/docker/compose/pull/1780). Also, I'm extremely passionate about this project and it would be a personal milestone for me to have contributed to the community for the first time (if that means anything which it might not). I would give anything to be the one to help with this.